dev/vcpkg/init.sh

```diff
@@ -26,6 +27,10 @@ for arg in "$@"; do
     ENABLE_HDFS=("${arg#*=}")
     shift # Remove argument name from processing
```

Comment:
does allow people enable flag both `ENABLE_HDFS` and `ENABLE_HDFS3` ?

Reply:
If we can't see performance gain from compile time link, we will close the PR.